connaisseur/alert.py

```diff
@@ -2,9 +2,10 @@
 import logging
 import os
 from datetime import datetime
+from typing import Optional
```

Comment:
That's probably something that can be debated whether we want to include it at all